persistence/src/main/java/edu/unc/lib/dl/persist/services/move/MoveObjectsJob.java

```diff
@@ -141,10 +144,52 @@ private void moveObject(PID objPid) {
         PID sourcePid = moveContent.getParent().getPid();
         addPidToSource(objPid, sourcePid);
 
+        // Write a premis event if object moves between admin units
+        adminUnitMove(objPid);
+
         // Add the object to its destination, which clears the previous parent as well
         destContainer.addMember(moveContent);
     }
 
+    private void adminUnitMove(PID sourcePid) {
+        ObjectPath destPath = objectPathFactory.getPath(destinationPid);
+        String destAdminUnit = null;
+        if (destPath != null) {
+            List<ObjectPathEntry> destObjPath = destPath.getEntries();
+            if (destObjPath.size() > 1) {
+                destAdminUnit = destObjPath.get(1).getPid();
+            }
+        }
+
+        ObjectPath sourcePath = objectPathFactory.getPath(sourcePid);
+        String currentAdminUnit = null;
+        if (sourcePath != null) {
+            List<ObjectPathEntry> objPath = sourcePath.getEntries();
+            if (objPath.size() > 1) {
+                currentAdminUnit = sourcePath.getEntries().get(1).getPid();
+            }
+        }
+
+        if (currentAdminUnit != null && destAdminUnit != null && !currentAdminUnit.equals(destAdminUnit)) {
+            RepositoryObject currentObj = repositoryObjectLoader.getRepositoryObject(sourcePid);
+            RepositoryObject parent = currentObj.getParent();
+            String prevContainerTitle = parent.getResource()
+                    .getProperty(DcElements.title).getObject().toString();
+
+            String newContainerUUID = destinationPid.getUUID();
+            String newContainerTitle = destContainer.getResource()
+                    .getProperty(DcElements.title).getObject().toString();
+
+            currentObj.getPremisLog().buildEvent(Premis.MetadataModification)
+                    .addAuthorizingAgent(agent.getUsername())
+                    .addEventDetail("Object moved from parent object {0}, " +
+                                    "{1} in Admin Unit {2} to {3}, {4} in Admin Unit {5}",
```

Comment:
Based off the discussion with anna, lets go with the following text:
`Object moved from source "<src title>" (<src id>) in Admin Unit "<src unit title>" (<src unit id>) to destination "<dest title>" (<dest id>) in Admin Unit "<dest unit title>" (<dest unit id>)`